ExcelWeb/Views/Home/Index.cshtml

```diff
@@ -3,6 +3,6 @@
 }
 
 <div class="text-center">
-    <h1 class="display-4">Welcome</h1>
-    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
+    <h1 class="display-4">API-only Project</h1>
```

Comment:
Leave useful message here now, we want to have a page so that if someone visits they can't do anything, but STRICTLY having API-only isn't so useful